Dell/recovery_backend.py

```diff
@@ -980,6 +980,21 @@ def create_ubuntu(self, recovery, revision, iso, platform, no_update, sender=Non
         xorrisoargs.append('-m')
         xorrisoargs.append(os.path.join('casper', old_initrd))
 
+        #Renew .disk/ubuntu_dist_channel for ubuntu-report
+        ubuntu_dist_channel = os.path.join(mntdir, '.disk', 'ubuntu_dist_channel')
+        if os.path.exists(ubuntu_dist_channel) and platform and revision:
+            xorrisoargs.append('-m')
+            xorrisoargs.append(ubuntu_dist_channel)
+            target = open(os.path.join(tmpdir, '.disk', 'ubuntu_dist_channel'), 'w')
```

Comment:
As a matter of preference, I would rather see this as `with open(os.path.join(tmpdir, '.disk', 'ubuntu_dist_channel'), 'w') as target` rather than `open()/target.close`.

Can you adjust that?  Otherwise this PR looks good to me.

Reply:
I have updated the commit. Could you help to check it again?